pkg/migrations/sql/versions/dev/0.3.1-dev/1-fix_permissions.sql

```diff
@@ -1,4 +1,6 @@
-CREATE OR REPLACE PROCEDURE execute_everywhere(command_key text, command TEXT, transactional BOOLEAN = true)
+DROP PROCEDURE execute_everywhere(text, TEXT, BOOLEAN);
```

Comment:
Need if exists just in case?

Reply:
(I'm not sure why we need to drop rather than create or replace here?

Reply:
will add the if exists. We are moving to a different schema which is why we drop the old